README.md

```diff
@@ -1471,6 +1472,21 @@ One consequence of this is that if you have an empty string in your `enum` array
 
 If you want to have the field set to a default value when empty you can provide a `ui:emptyValue` field in the `uiSchema` object.
 
+## Localize default messages
+
+You can change localize to default messages errors.
+ 
+Here are some examples from the [playground](http://mozilla-services.github.io/react-jsonschema-form/).
+ 
+Here list of languages support [ajv-i18n](https://github.com/epoberezkin/ajv-i18n).
```

Comment:
Here is the list of supported languages [ajv-i18n](https://github.com/epoberezkin/ajv-i18n).